templates/include/telemetry/environment.1.schema.json

```diff
@@ -10,25 +10,43 @@
             "type": "object",
             "properties": {
               "blocklisted": {
-                "type": "boolean"
+                "type": "boolean",
+                "description": "Whether or not the add-on appears in the blocklist. This field is only available after the 'sessionstore-windows-restored' topic is notified."
               },
               "description": {
-                "type": [ "string", "null" ]
+                "type": [
+                  "string",
+                  "null"
+                ]
               },
               "name": {
-                "type": "string"
+                "type": "string",
+                "description": "The add-on name, limited to 100 characters. This field is only available after the 'sessionstore-windows-restored' topic is notified."
               },
               "userDisabled": {
-                "type": [ "boolean", "integer" ]
+                "type": [
+                  "boolean",
+                  "integer"
+                ],
+                "description": "Whether or not the user disabled the add-on. This field is only available after the 'sessionstore-windows-restored' topic is notified."
               },
               "appDisabled": {
-                "type": "boolean"
+                "type": "boolean",
+                "description": "True if this add-on cannot be used in the application based on version compatibility, dependencies, and blocklisting. This field is only available after the 'sessionstore-windows-restored' topic is notified."
               },
               "version": {
-                "type": [ "string", "number" ]
+                "type": [
+                  "string",
+                  "number"
+                ],
+                "description": "The version of the add-on. This field is available at startup."
               },
               "scope": {
-                "type": [ "integer", "null" ]
+                "type": [
+                  "integer",
+                  "null"
+                ],
```

Comment:
Writing the JSON with descriptions to a new file changed the formatting for some fields. However, it seems there is no real formatting standard right now. so it's probably fine?

Reply:
This seems fine to me. Indeed, there's no style constraints on templates and includes, though the generated schemas have consistent formatting enforced. It's pretty easy to see from the generated schemas here that this PR is correctly just adding descriptions.

Reply:
Seconded -- it's actually kind of nice to have the templates formatted too, but it doesn't need to be a sweeping change since the generated schemas are formatted.